fenix/app/metrics.yaml

```diff
@@ -9087,6 +9085,25 @@ client_deduplication:
       - android-probes@mozilla.com
       - fbertsch@mozilla.com
     expires: 122
+  advertising_id:
+    type: string
+    lifetime: ping
+    description: |
+      A hashed and salted version of the Google Advertising ID from the device.
+      This is its own metric because we send a differently salted version of
+      the ad ID two separate pings.
```

Comment:
I don't thing this is needed here, but might be needed in the commit message.

Reply:
Done